lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py

```diff
@@ -133,181 +146,194 @@ def __init__(self):
             self.module.fail_json(msg="the python NetApp-Lib module is required")
         else:
             self.server = netapp_utils.setup_na_ontap_zapi(module=self.module)
+
+        if HAS_IPADDRESS_LIB is False:
+            self.module.fail_json(msg="the python ipaddress lib is required for this module")
```

Comment:
@carchi8py if you could document this as a requirement for this module, that would be great. 

also, for future use, in 2.8 we added 'missing_required_lib` so it would look something like `self.module.fail_json(msg=missing_required_lib("ipaddress")`.

Reply:
Added a line for the Requirements to the documentation.

Reply:
+1